docs/generate-time-series/go.rst

```diff
@@ -0,0 +1,374 @@
+
+.. _gen-ts-go:
+
+==================================
+Generate time series data using Go
+==================================
+
+This tutorial will show you how to generate some :ref:`experimental time 
+series data <gen-ts>` from information about the `International Space Station`_
+using `Go`_.
+
+.. SEEALSO::
+
+    :ref:`gen-ts`
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+Prerequisites
+=============
+
+You must have CrateDB :ref:`installed and running <install-run>`.
+
+Make sure you're running an up-to-date version of `Go`_. We recommend 
+Go 1.11 or higher since we will be making use of `modules`_. 
```

Comment:
can you reword this (and maybe other places?) to use the second person instead of the first person? cf. https://developers.google.com/style/person